frontend/src/features/mapSideBar/SidebarContents/subforms/ClassificationForm.test.tsx

```diff
@@ -1,29 +1,128 @@
+  { value: Classifications.CoreStrategic, label: 'Core Strategic' } as SelectOption,
+  { value: Classifications.SurplusEncumbered, label: 'Surplus Encumbered' } as SelectOption,
+] as SelectOptions;
+
+const mockKeycloak = (claims: string[]) => {
+  (useKeycloak as jest.Mock).mockReturnValue({
+    keycloak: {
+      subject: 'test',
+      userInfo: {
+        roles: claims,
+        agencies: ['1'],
+      },
+    },
+  });
+};
+const mockStore = configureMockStore([thunk]);
+let history = createMemoryHistory();
+
+const lCodes = {
+  lookupCodes: [
+    { name: 'agencyVal', id: '1', isDisabled: false, type: API.AGENCY_CODE_SET_NAME },
+    { name: 'disabledAgency', id: '2', isDisabled: true, type: API.AGENCY_CODE_SET_NAME },
+    { name: 'roleVal', id: '1', isDisabled: false, type: API.ROLE_CODE_SET_NAME },
+    { name: 'disabledRole', id: '2', isDisabled: true, type: API.ROLE_CODE_SET_NAME },
+    {
+      name: 'Core Operational',
+      id: '0',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+    {
+      name: 'Core Strategic',
+      id: '1',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+    {
+      name: 'Surplus Active',
+      id: '2',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+    {
+      name: 'Surplus Encumbered',
+      id: '3',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+    {
+      name: 'Disposed',
+      id: '4',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+    {
+      name: 'Demolished',
+      id: '5',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+    {
+      name: 'Subdivided',
+      id: '6',
+      isDisabled: false,
+      type: API.PROPERTY_CLASSIFICATION_CODE_SET_NAME,
+    },
+  ] as ILookupCode[],
+};
+
+const getStore = () =>
+  mockStore({
+    [reducerTypes.PROPERTY_NAMES]: ['test'],
+    [reducerTypes.LOOKUP_CODE]: lCodes,
+  });
 
 it('renders correctly', () => {
   const tree = renderer.create(form).toJSON();
   expect(tree).toMatchSnapshot();
 });
 
 const form = (
-  <Formik onSubmit={noop} initialValues={{ classificationId: '' }}>
-    <ClassificationForm classifications={mockClassifications} field="classificationId" />
-  </Formik>
+  <Provider store={getStore()}>
+    <Router history={history}>
+      <Formik onSubmit={noop} initialValues={{ classificationId: '' }}>
+        <ClassificationForm
+          classifications={mockClassifications}
+          field="classificationId"
+          encumbranceField="test"
+        />
+      </Formik>
+    </Router>
+  </Provider>
 );
 
 describe('renders definitions correctly', () => {
+  mockedAxios.get.mockImplementationOnce(() => Promise.resolve({}));
+
+  beforeEach(() => {
+    mockKeycloak([]);
+  });
   afterEach(() => {
     cleanup();
     jest.clearAllMocks();
```

Comment:
this might clear your axios mock above, just be careful.

Reply:
remember that the order that tests are run in isn't guaranteed, so if more then one test needs the axios mock your tests may pass right now but break later when the execution order changes.